### PR TITLE
fruity: Add ProductType to query_system_parameters

### PR DESCRIPTION
**Pull request by as0ler**

### CODE REVIEW

File: src/fruity/fruity-host-session.vala [hunk 1]
```diff
@@ -623,6 +623,7 @@ namespace Frida {
 				os["id"] = "ios";
 				os["name"] = properties.get_string ("ProductName");
 				os["version"] = properties.get_string ("ProductVersion");
+				os["type"] = properties.get_string ("ProductType");
```
Comment (oleavr) [inline src/fruity/fruity-host-session.vala:626]: I'm thinking this belongs on the parent, as it's not a property of the operating system, but the system itself. Perhaps we should name it `model`, if that makes sense?

**Review by oleavr — CHANGES_REQUESTED** [review]

File: src/fruity/fruity-host-session.vala (hunk 1 above)
Comment (as0ler) [reply src/fruity/fruity-host-session.vala]: Make sense. Updated!

**Review by oleavr — CHANGES_REQUESTED** [review]
Thanks! 🤘  It seems `HWModelStr` is missing.  Could you give an example from `ideviceinfo` on a device where you're seeing such a value?  Wondering if it has a different name outside of MobileGestalt.

**Comment by as0ler** [comment]
Adding HardwareModel into hardware model key.

**Pull request merged**